### PR TITLE
type hint user command example

### PR DESCRIPTION
**Pull request by Rxann**
## Summary

This pull request type hints the user command example on the readme file of the documentation. I feel this example needs to be type hinted to avoid errors with users using the code.

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Comment by Grace-codes** [comment]
The correct typehint should be `Union[discord.User, discord.Member]`

**Comment by nerdguyahmad** [comment]
> The correct typehint should be `Union[discord.User, discord.Member]`

README examples should be kept simple and `discord.Member` seems fine here.

File: README.rst [hunk 1]
```diff
@@ -80,7 +80,7 @@ Quick Example
         await ctx.send(f"Hello {name}!")
         
     @bot.user_command(name="Say Hello")
-    async def hi(ctx, user : discord.User):
+    async def hi(ctx, user : discord.Member):
```
Comment (nerdguyahmad) [inline README.rst:83]: Remove the whitespace between "user" and ":". See [this](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements)

**Review by nerdguyahmad — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (Rxann) [reply README.rst:83]: Like this 

user:discord.Member?

Comment (nerdguyahmad) [reply README.rst:83]: No, Follow the PEP-8 code styling conventions.
```
user: discord.Member
```

Comment (Rxann) [reply README.rst]: Alr ty, I just commited the change

**Review by nerdguyahmad — APPROVED** [review]
Thanks.

**Review by Grace-codes — APPROVED** [review]

**Comment by BobDotCom** [comment]
For consistency purposes, this is unnecessary. If this was added, it would raise a question as to why `ctx` was not typehinted to `discord.ApplicationContext` and we would fall into a rabbit hole that would lead to considerably more confusing code, defeating the purpose of an example. Best to not add this for simplistic and consistency purposes.

**Pull request closed without merging**